src/vmm/src/arch/aarch64/fdt.rs

```diff
@@ -589,7 +589,7 @@ mod tests {
         // let mut output = fs::OpenOptions::new()
         // .write(true)
         // .create(true)
-        // .open(path.join(format!("src/aarch64/{}", dtb_path)))
+        // .open(path.join(format!("src/arch/aarch64/{}", dtb_path)))
```

Comment:
how do we create DTBs btw? What are the sources?

Reply:
it is an abomination :P We uncomment the snippet you commented on, run the test so that it creates the DTB then comment it out again. I am actually trying to thing if it makes sense to carve out these little scripts in small binaries within the crate.